README.md

```diff
@@ -488,6 +488,15 @@ Response Code(s)
 |---|------------------|
 | 200 | Default response |
 
+#### Get Client's suspension status
```

Comment:
Maybe this should be more like `Get Agent's suspension status on behalf of Clients`, basically it's still the agent we are checking while a client is the logged in party and not the client's suspension status

Reply:
Done